thirdeye-persistence/src/test/java/ai/startree/thirdeye/datalayer/bao/TestAnomalyManager.java

```diff
@@ -183,19 +185,48 @@ public void testFeedback() {
     Assert.assertEquals(mergedResult1.getFeedback().getFeedbackType(), AnomalyFeedbackType.ANOMALY);
   }
 
+  @Test
+  public void testGetAllAnomaliesForFeedback() {
+    final Long id1 = mergedAnomalyResultDAO.save(buildAnomaly(10000, 11000));
+    final Long id2 = mergedAnomalyResultDAO.save(buildAnomaly(12000, 13000));
+
+    final AnomalyDTO anomalyForFeedback = mergedAnomalyResultDAO.findById(id1);
```

Comment:
Have a method `createAnomalyWithFeedback` here that you can use in both tests

Reply:
Created `provideFeedbackToAnomaly` instead to provide flexibility on using existing anomalies for feedback.